source/includes/_account.md

```diff
@@ -5484,4 +5484,94 @@ $accountTimeline = $apiInstance->getAccountTimeline($accountID);
 
 **Response**
 
-If successful, returns a status code of 200 and a complete account record including: the account object; bundles with subscriptions and timelines giving all events; invoices; and payments including payment attempts.
\ No newline at end of file
+If successful, returns a status code of 200 and a complete account record including: the account object; bundles with subscriptions and timelines giving all events; invoices; and payments including payment attempts.
+
+## CBA Rebalancing
+
+This API endpoint allows to rebalance the existing credit balance adjustments against the unpaid invoices in the Account.
```

Comment:
Let's try to be consistent on how we present the 'endpoints'. I see for instance [here](https://killbill.github.io/slate/account.html#retrieve-account-invoices), we state `This endpoint provides an API...` (and not 'This API endpoint...')

Reply:
`This API endpoint allows to rebalance the existing credit balance adjustments against the unpaid invoices in the Account.`

-> `This endpoint is used to apply existing account credit to unpaid invoices (if any). Normally the system does this automatically, so there is rarely a need to invoice such endpoint.`

Reply:
Hi @sbrossie added these changes.